packages/lib/services/interop/InteropService_Importer_Md_frontmatter.ts

```diff
@@ -12,6 +12,9 @@ export default class InteropService_Importer_Md_frontmatter extends InteropServi
 			const { metadata, tags } = parse(note.body);
 
 			const updatedNote = { ...note, ...metadata };
```

Comment:
I think it's better if the object is fully set from the start, rather than tweak it afterwards

```suggestion
			const updatedNote = {
				...note,
				...metadata,
				title: metadata.title ? metadata.title : note.title,
			};
```